models/aeif_cond_alpha.h

```diff
@@ -84,51 +86,51 @@ This implementation uses the embedded 4th order Runge-Kutta-Fehlberg solver with
 adaptive step size to integrate the differential equation.
 
 The membrane potential is given by the following differential equation:
-C dV/dt= -g_L(V-E_L)+g_L*Delta_T*exp((V-V_T)/Delta_T)-g_e(t)(V-E_e)
-                                                     -g_i(t)(V-E_i)-w +I_e
+@f[ C dV/dt= -g_L(V-E_L)+g_L*Delta_T*exp((V-V_T)/Delta_T)-g_e(t)(V-E_e)
+                                                     -g_i(t)(V-E_i)-w +I_e @f]
 
 and
 
-tau_w * dw/dt= a(V-E_L) -W
+@f[ tau_w * dw/dt= a(V-E_L) -W @f]
 
 Parameters:
```

Comment:
In the rendered version, the headings and subheadings don't seem to be handled as headings, but rather as normal text. Is there any way to mark them up as proper headings at different levels?

Reply:
Unfortunately, [breathe](https://github.com/michaeljones/breathe) replaces the heading tags with just emphasized text. e.g. source file:
`## Parameters` 
generated HTML: 
` <p> <em> Parameters: </em> </p>`   
So to make headings stand out we could add bold or italics.

Reply:
is this also true when using rst-style headers? like 

```
title
=====

subtitle
--------

subsubtitle
+++++++++++
```

Reply:
Yes. I've tried a few things but nothing worked. The XML file still has the heading tags  `<para><heading level="1">Parameters: </heading> </para>` but this is removed in HTML.